rotkehlchen/chain/bitcoin/__init__.py

```diff
@@ -9,62 +7,102 @@
 from rotkehlchen.utils.network import request_get_dict
 
 
-def _have_bc1_accounts(accounts: List[BTCAddress]) -> bool:
+class BchNotSupported(Exception):
+    ...
+
+
+def _have_bc1_accounts(accounts: list[BTCAddress]) -> bool:
```

Comment:
please do not change the consistency of the code by switching to the built-in `list` for type hints instead of the `List`. Yes since python 3.9 this is possible but we would first need to discuss if we want this change or not. If we do we can begin doing the change slowly over the code.

But don't go and change consistency by introducing it.